modules/auth0/tenant/variables.tf

```diff
@@ -0,0 +1,110 @@
+variable "region" {
+  type        = string
+  description = "AWS Region"
+}
+
+variable "friendly_name" {
+  type        = string
+  description = "The friendly name of the Auth0 tenant. If not provided, the module context ID will be used."
+  default     = ""
+}
+
+variable "picture_url" {
+  type        = string
+  description = "The URL of the picture to be displayed in the Auth0 Universal Login page."
+  default     = "" # TODO provide a default here
```

Comment:
See TODO

Reply:
I'd suggest either linking to the Cloud Posse logo, probably

Reply:
added a logo